department-of-veterans-affairs/caseflow
# Mkerns/caseflow 2182 complete task

Resolves [CASEFLOW-2182](https://vajira.max.gov/browse/CASEFLOW-2182)

### Description
Adds the on hold task action to the AssessDocumentationTask, allowing either VhaProgramOffice or VhaRegionalOffice users to place their task on hold.

### Acceptance Criteria

### Testing Plan
1. Log in as a BvaIntake user, and complete a 10182 (Appeal) Intake with at least one VHA issue 
2. Log in as a CAMO user and assign the appeal to the Program Office
3. Log in as a Program Office user and attempt to complete task.
4. Check the appeal's task tree to confirm the tasks are in the correct state
5. Confirm the task is marked as complete, and the parent CAMO task is assigned

### User Facing Changes
